### PR TITLE
Docs on how to package a release + CHANGELOG for 0.24.0

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -415,3 +415,62 @@ https://github.com/apache/incubator-superset/pull/3013
...
+    # check travis to confirm the build succeeded as
+    # you shouldn't assume that a clean cherry will be clean
+    # when landing on a new sundae
+
+    # compile the JS, and push to pypi
+    ./pypi_push.sh
+
+    # publish an update to the CHANGELOG.md for the right version range
+    # looking the latest CHANGELOG entry for the second argument
+    ./gen_changelog.sh 0.25.0 0.22.1
```
Comment (xrmx) [inline CONTRIBUTING.md]: versions looks reversed, also as an example something like 0.24 0.25 would be easier to follow

Comment (mistercrunch) [reply CONTRIBUTING.md]: Following the logic in the underlying tool. I don't feel strongly about it either way. npm's github-changes

Comment (xrmx) [reply CONTRIBUTING.md]: From gen_changelog.sh:
```
# usage:  ./github-changes 0.20.0 0.20.1
# will overwrites the local CHANGELOG.md, somehow you need to merge it in
github-changes -o apache -r incubator-superset --token $GITHUB_TOKEN --between-tags $1...$2
```

So $1 should be older than $2

Comment (mistercrunch) [reply CONTRIBUTING.md]: Oh you're right!

**Review by fabianmenges — APPROVED** [review]

**Pull request merged**